### PR TITLE
Skip noarch linting for cnvkit

### PR DESCRIPTION
**Pull request by micknudsen**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [X] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

See #15747

### CODE REVIEW

File: recipes/cnvkit/meta.yaml [hunk 1]
```diff
@@ -48,3 +48,5 @@ extra:
   identifiers:
     - biotools:cnvkit
     - doi:10.1371/journal.pcbi.1004873
+  skip-lints:
+    - should_be_noarch_generic
```
Comment (dpryan79) [inline recipes/cnvkit/meta.yaml]: Why isn't this `noarch: generic`? Does conda complain due to the selector for futures then?

Comment (micknudsen) [reply recipes/cnvkit/meta.yaml]: To be honest, I don't fully understand how this `noarch` thing works. I just noticed that when I used it here (to satisfy the linter, see https://github.com/bioconda/bioconda-recipes/pull/15717), it only generated a Python 2.7 build.

Comment (dpryan79) [reply recipes/cnvkit/meta.yaml]: I've set it to `noarch: python`, we'll see if conda-build complains about the selector in the run section.

Comment (dpryan79) [reply recipes/cnvkit/meta.yaml]: Looks like it's happy :)

**Comment by dpryan79** [comment]
@bioconda-bot merge

**Comment by micknudsen** [comment]
@dpryan79 Thanks!

**Pull request merged**